transport.go

```diff
@@ -0,0 +1,41 @@
+package gidari
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/alpine-hodler/gidari/internal/transport"
+)
+
+// Config is the configuration object used to make programatic Transport requests.
+type Config struct {
+	transport.Config
+}
+
+// TransportFile will construct the transport operation using a configuration YAML file.
+func TransportFile(ctx context.Context, file *os.File) error {
+	bytes := make([]byte, 437)
+
+	count, err := file.Read(bytes)
+	if err != nil {
+		return err
+	}
+
+	log.Printf("read %d bytes from %s", count, file.Name())
+
+	cfg, err := transport.NewConfig(bytes)
```

Comment:
@prestonvasquez but aren't we already using transport.NewConfig here?

Reply:
Yeah we are, that's okay since it's not user-accessible. We just need a way for a user to build a request since the internal package is not importable.